charts/timesketch/values-production.yaml

```diff
@@ -0,0 +1,280 @@
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
+  affinity: {}
+## Persistence Storage Parameters
+##
+persistence:
+  ## @param persistence.name Timesketch persistent volume name
+  ##
+  name: timesketchvolume
+  ## @param persistence.size Timesketch persistent volume size
+  ##
+  size: 1T
+  ## @param persistence.storageClass PVC Storage Class for Timesketch volume
+  ## If default, storageClassName: <storageClass>, which enables GCP Filestore
+  ##   when using the Filestore CSI Driver
+  ##   ref https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/filestore-csi-driver#access
+  ## If set to "-", storageClassName: "", which disables dynamic provisioning
+  ## If undefined or set to null, no storageClassName spec is
+  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+  ##   GKE, AWS & OpenStack)
+  ## ref https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/#using-dynamic-provisioning
+  ## 
+  storageClass: standard-rwx
+  ## @param persistence.accessModes PVC Access Mode for Timesketch volume
+  ## Access mode may need to be updated based on the StorageClass
+  ## ref https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes
+  ## 
+  accessModes:
+    - ReadWriteMany
+## @section Third Party Configuration
+## This section contains all the main configuration for third party dependencies
+## Timesketch requires to run
+##
+## @section Opensearch Configuration Parameters
+## IMPORTANT: The Opensearch Security Plugin / TLS has not yet been configured by default
+## ref on steps required https://opensearch.org/docs/1.1/security-plugin/configuration/index/
+## To see a full list of available values, run helm show values charts/opensearch*
+##
+opensearch:
+  ## @param opensearch.enabled Enables the Opensearch deployment
+  ##
+  enabled: true
+  ## @param opensearch.replicas Number of Opensearch instances to deploy
+  ##
+  replicas: 3
+  ## @param opensearch.sysctlInit.enabled Sets optimal sysctl's through privileged initContainer
+  ##
+  sysctlInit:
+    enabled: true
+  ## @param opensearch.opensearchJavaOpts Sets the size of the Opensearch Java heap
+  ## It is recommended to use at least half the system's available ram
+  ## 
+  opensearchJavaOpts: "-Xms32g -Xmx32g"
+  ## @param opensearch.httpPort Opensearch service port
+  ##
+  httpPort: 9200
+  ## Opensearch persistence configuration
+  ##
+  persistence:
+    ## @param opensearch.persistence.size Opensearch Persistent Volume size. A persistent volume would be created for each Opensearch replica running
+    ##
+    size: 1Ti
+  ## Opensearch resource requests
+  ## @param opensearch.resources.requests.cpu Requested cpu for the Opensearch containers
+  ## @param opensearch.resources.requests.memory Requested memory for the Opensearch containers 
+  ##
+  resources:
+    requests:
+      cpu: 8000m
+      memory: 32Gi
```

Comment:
We set `opensearchJavaOpts: "-Xms32g -Xmx32g"` which should be half of the available memory. Should we set this one to 64Gi?

Reply:
Ah agreed, updated.